docs/reference/remap.cue

```diff
@@ -65,4 +64,205 @@ remap: {
 			description: "Raised when the provided input cannot be parsed."
 		}
 	}
+
+	// TRL type system
+	types: [TypeName=string]: {
+		#Use: "parameter" | "return"
+
+		description: string
+		use: [#Use, ...#Use]
+	}
+
+	types: {
+		"any": {
+			description: "A stand-in for any type."
+			use: ["parameter"]
+		}
+		"array": {
+			description: "A list of items."
+			use: ["parameter", "return"]
+		}
+		"boolean": {
+			description: "`true` or `false`."
+			use: ["parameter", "return"]
+		}
+		"float": {
+			description: "A 64-bit floating-point number."
+			use: ["parameter", "return"]
+		}
+		"map": {
+			description: """
+				A key-value map in which keys are strings and values can be of any TRL type,
+				including maps.
+				"""
+			use: ["parameter", "return"]
+		}
+		"integer": {
+			description: "A 64-bit integer."
+			use: ["parameter", "return"]
+		}
+		"null": {
+			description: "No value."
+			use: ["return"]
```

Comment:
Are you sure `null` can't be passed in as a function argument? If that's the case, then that's definitely a bug 🤔